_docs/compliance/logging-requirements.md

```diff
@@ -16,8 +16,7 @@ This document outlines how our platform enables cloud.gov customers to comply wi
 
 ### Minimum Logging Data
 
-Cloud.gov ensures that platform component and application logs contain the minimum
-required data where possible.
+Cloud.gov ensures that platform component and application logs contain the minimum required data where possible. Customers are responsible for logging any custom/additional HTTP headers at the application level.
```

Comment:
Do you think we should note the headers that we do log by default so that customers know what additional headers they are responsible for?

Reply:
Not sure. Might be good to create a new ticket for that.